ui/packages/shared/profile/src/ProfileIcicleGraph/index.tsx

```diff
@@ -176,13 +180,41 @@ const ProfileIcicleGraph = function ProfileIcicleGraphNonMemo({
     // Do necessary checks to ensure that icicle chart can be rendered for this query.
     if (isInvalidIcicleChartQuery) {
       if (isNonDelta) {
-        return <ErrorContent errorMessage="Icicle chart is only available for delta profiles." />;
+        return (
+          <ErrorContent
+            errorMessage={
+              <>
+                <span>To use the Icicle chart, please switch to a Delta profile.</span>
```

Comment:
Should we maybe just not allow people to select it in those cases in the first place?

Reply:
UX wise I feel this would be better, just not allowing them to select could leave them puzzled on 'why?'.